yt_dlp/extractor/nrk.py

```diff
@@ -55,11 +55,13 @@ def _raise_error(self, data):
         message = data.get('endUserMessage') or MESSAGES.get(message_type, message_type)
         raise ExtractorError(f'{self.IE_NAME} said: {message}', expected=True)
 
-    def _call_api(self, path, video_id, item=None, note=None, fatal=True, query=None):
+    def _call_api(self, path, video_id, item=None, note=None, fatal=True, query=None, headers=None):
```

Comment:
if every API call is currently using the new header then we don't need to add a param
```suggestion
    def _call_api(self, path, video_id, item=None, note=None, fatal=True, query=None):
```